features/admin.users.v1/components/user-groups-list.tsx

```diff
@@ -0,0 +1,493 @@
+                        onClick={ handleOpenAddNewGroupModal }
+                    >
+                        <Icon name="plus"/>
+                        { t("applicationRoles:roleGroups.assignGroup") }
+                    </PrimaryButton>
+                ) }
+            </Box>
+            <Divider hidden/>
+            <ListLayout
+                advancedSearch={ (
+                    <AdvancedSearchWithBasicFilters
+                        data-testid="group-mgt-groups-list-advanced-search"
+                        onFilter={ (query: string) => setSearchQuery(query?.trim()) }
+                        disableSearchFilterDropdown
+                        filterAttributeOptions={ [] }
+                        placeholder={ t("console:manage.features.groups.advancedSearch.placeholder") }
+                        defaultSearchAttribute={ "" }
+                        defaultSearchOperator={ "" }
+                        triggerClearQuery={ triggerClearQuery }
+                    />
+                ) }
+                currentListSize={ paginatedGroups?.length }
+                listItemLimit={ listItemLimit }
+                onItemsPerPageDropdownChange={ handleItemsPerPageDropdownChange }
+                data-componentid={ `${ componentId }-user-groups-list-layout` }
+                data-testid={ `${ componentId }-user-groups-list-layout` }
+                onPageChange={ handlePaginationChange }
+                showPagination={ true }
+                totalPages={ resolveTotalPages() }
+                totalListSize={ userGroupsList?.length }
+                showPaginationPageLimit={ !isReadOnly }
+                showTopActionPanel={ (!!searchQuery || userGroupsList?.length > 0) }
+            >
+                <DataTable<GroupsMemberInterface>
+                    isLoading={ isLoading }
+                    loadingStateOptions={ {
+                        count: UIConstants.DEFAULT_RESOURCE_LIST_ITEM_LIMIT,
+                        imageType: "circular"
+                    } }
+                    actions={ resolveTableActions() }
+                    columns={ resolveTableColumns() }
+                    data={ paginatedGroups }
+                    onRowClick={ () =>  null }
+                    placeholders={ showPlaceholders() }
+                    selectable={ false }
+                    showHeader={ true }
+                    data-testid={ componentId }
+                />
+            </ListLayout>
+            <ConfirmationModal
+                primaryActionLoading={ isDeleting }
+                data-testid={ `${ componentId }-confirmation-modal` }
+                data-componentid={ `${ componentId }-confirmation-modal` }
+                onClose={ (): void => setShowDeleteConfirmationModal(false) }
+                type="negative"
+                open={ showDeleteConfirmationModal }
+                primaryAction="Confirm"
+                secondaryAction="Cancel"
```

Comment:
Shall we use i18n here?

```suggestion
                primaryAction="Confirm"
                secondaryAction="Cancel"
```

Reply:
Fixed with cb0ee2cf165319ca096aa44d358799060f08bbf9